hazelcast/src/test/java/com/hazelcast/test/jdbc/MySQLDatabaseProvider.java

```diff
@@ -20,11 +20,14 @@
 
 public class MySQLDatabaseProvider implements TestDatabaseProvider {
 
+    private static final int LOGIN_TIMEOUT = 5;
```

Comment:
This is short - it might be downloading a docker image, we should allow something much longer - 120 s maybe.